feeder.php

```diff
@@ -74,4 +75,85 @@ function hookHeader($params)
         ]);
         return $this->display(__FILE__, 'feederHeader.tpl');
     }
+
+    private function initDefaultConfigurationValues()
+    {
+        if (!Configuration::hasKey('FEEDER_INCLUDE_IMAGES')) {
+            Configuration::updateValue('FEEDER_INCLUDE_IMAGES', 1);
+        }
+
+        return true;
+    }
+
+    public function getContent()
+    {
+        $output = null;
+
+        if (Tools::isSubmit('submit' . $this->name)) {
+            $value = strval(Tools::getValue('FEEDER_INCLUDE_IMAGES'));
+            Configuration::updateValue('FEEDER_INCLUDE_IMAGES', $value);
+            $output .= $this->displayConfirmation($this->l('Settings updated'));
+        }
+        return $output . $this->displayForm();
+    }
+
+    public function displayForm()
+    {
+        // Get default language
+        $default_lang = (int)Configuration::get('PS_LANG_DEFAULT');
+
+        // Init Fields form array
+        $fields_form[0]['form'] = [
```

Comment:
This variable is not initiliazed

Reply:
I initialized the array (c8a8082). I will rebase my commits after the full review